src/arabic.c

```diff
@@ -220,27 +218,26 @@ chg_c_laa2i(int hid_c)
 /*
  * Change shape - from Combination-Isolated to Final
  */
-    static int
+static int
 chg_c_laa2f(int hid_c)
 {
     int tempc;
 
-    switch (hid_c)
-    {
-	case a_ALEF_MADDA:
-	    tempc = a_f_LAM_ALEF_MADDA_ABOVE;
-	    break;
-	case a_ALEF_HAMZA_ABOVE:
-	    tempc = a_f_LAM_ALEF_HAMZA_ABOVE;
-	    break;
-	case a_ALEF_HAMZA_BELOW:
-	    tempc = a_f_LAM_ALEF_HAMZA_BELOW;
-	    break;
-	case a_ALEF:
-	    tempc = a_f_LAM_ALEF;
-	    break;
-	default:
-	    tempc = 0;
+    switch (hid_c) {
```

Comment:
The convention is to place the opening curly braces for the switch statement in a separate line.